### ISSUE TITLE
Deprecation warning with new ruamel version

### ISSUE THREAD

**Issue by LaurentBergeron** [CINPLA/exdir#139]
At multiple location in the `exdir` package, `yaml.safe_load` is used. This syntax will be deprecated by Ruamel. Using ruamel 0.17.21, the following Deprecation warning appears:
```
PendingDeprecationWarning:
  load will be removed, use

    yaml=YAML(typ='safe', pure=True)
    yaml.load(...)
```

We should update this code base to use the new syntax

**Comment by berquist** [comment]
I think I fixed this in #152.

**Comment by dragly** [comment]
I think you are right. Closing the issue :)
